test/unit/cog_api/fake/users_test.exs

```diff
@@ -0,0 +1,42 @@
+defmodule CogApi.Fake.UsersTest do
+  use CogApi.FakeCase
+
+  alias CogApi.Fake.Users
+
+  doctest CogApi.Fake.Users
+
+  describe "index" do
+    it "returns a list of users" do
+      params = %{
+        first_name: "Leo",
+        last_name: "McGary",
+        email_address: "cos@example.com",
+        username: "chief_of_staff",
+        password: "supersecret",
+      }
+      {:ok, _} = Users.create(fake_endpoint, params)
+
+      {:ok, users} = Users.index(fake_endpoint)
+
+      last_user = List.last users
+      assert present last_user.id
+      assert last_user.first_name == params.first_name
+    end
+  end
+
+  describe "create" do
+    it "returns the created user" do
+      params = %{
+        first_name: "Leo",
+        last_name: "McGary",
+        email_address: "cos@example.com",
+        username: "chief_of_staff",
+        password: "supersecret",
+      }
+      {:ok, user} = Users.create(fake_endpoint, params)
+
+      assert present user.id
+      assert user.first_name == params.first_name
```

Comment:
Should we be asserting all the other attributes here? This is our only check that it isn't just ignoring those other options.